packages/react-generator/src/generator/page.js

```
@@ -376,34 +383,82 @@ const generateReactCode = ({ schema, name, type, componentsMap }) => {
+
+  const stringUseEffect = `
+      useEffect(() => {
+        ${componentDidMount && componentDidMount['body'] ? componentDidMount['body'] : ''}
+        ${componentWillUnmount && componentWillUnmount['body'] ? `return () => {${componentWillUnmount['body']}}` : ''}
+        }, [${componentDidUpdate && componentDidUpdate['params'] ? componentDidUpdate['params'] : ''}])
+    `
+
+  const stringUseLayoutEffect = `
+         useLayoutEffect(() => {
+        ${componentWillMount && componentWillMount['body'] ? componentWillMount['body'] : ''}
+        }, [${componentDidUpdate && componentDidUpdate['params'] ? componentDidUpdate['params'] : ''}])
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)